package.json

```diff
@@ -40,6 +40,7 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
     "file-loader": "^6.0.0",
+    "flag-icons": "^6.1.1",
```

Comment:
I note this is not locked and we talked about how you might want to lock it, but if you're happy that's fine.